docs/preview/index.md

```diff
@@ -21,8 +21,9 @@ For more granular packages we recommend reading the documentation.
     - [Securely Receive CloudEvents](features/cloudevent/receive-cloudevents-job)
 - **Security**
     - [Automatically invalidate cached secrets from Azure Key Vault](features/security/auto-invalidate-secrets)
-- **Telemetry**
+- **Databricks**
     - [Measure Databricks job run outcomes as metric](features/databricks/job-metrics)
+    - [Interact with Databricks to gain insights](features/databricks/gain-insights)
```

Comment:
The current file is still called docs/preview/features/databricks/databricks-info-provider.md, can we rename it please?

Reply:
Yes, will do.